pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -390,29 +390,33 @@ contract StablePhantomPool is IRateProvider, BaseGeneralPool, ProtocolFeeCache {
         // Would like to use a function selector here, but it causes stack issues
         if (swapRequest.kind == IVault.SwapKind.GIVEN_IN) {
             return
-                _onSwapBptGivenIn(
-                    swapRequest.amount,
-                    indexIn,
-                    indexOut,
-                    bptIsTokenIn,
-                    amp,
-                    protocolSwapFeePercentage,
-                    virtualSupply,
-                    balancesWithoutBpt,
-                    scalingFactors
+                _downscaleDown(
```

Comment:
I'd add a comment mentioning that since we're returning an amount out, we downscale down (and the opposite in the other case)